hq_superset/api.py

```diff
@@ -36,8 +36,8 @@ def save_token(token, request):
     tok = Token(
         client_id=client.client_id,
         expires_at=expires_at,
-        access_token=token['access_token'],
-        token_type=token['token_type'],
+        access_token=token["access_token"],
```

Comment:
Absolutely not.

Reply:
You might find https://github.com/dimagi/hq_superset/pull/40/commits/a8404412fdaf309d3b20afa0efa99b78978be89c to your liking?